### PR TITLE
added new css for dark scheme #14833

### PR DESCRIPTION
**Pull request by maheshprajapati**
This issue resolved when this CSS added

### CODE REVIEW

File: includes/wc-template-functions.php [hunk 1]
```diff
@@ -1966,7 +1966,7 @@ function woocommerce_form_field( $key, $args, $value = null ) {
 
 					$field .= '<input type="hidden" class="hidden" name="' . esc_attr( $key ) . '" id="' . esc_attr( $args['id'] ) . '" value="" ' . implode( ' ', $custom_attributes ) . ' placeholder="' . esc_attr( $args['placeholder'] ) . '" />';
 
-				} elseif ( is_array( $states ) ) {
+				} elseif ( is_array(!empty( $states )) ) {
```
Comment (claudiosanches) [inline includes/wc-template-functions.php]: `empty()` returns boolean, so will make `is_array()` always return false.
And no idea why this change is required here.

File: assets/css/twenty-seventeen.scss [hunk 2]
```diff
@@ -826,6 +826,11 @@ button.pswp__button--zoom:hover {
 		content: "→";
 	}
 }
+.colors-dark {
```
Comment (claudiosanches) [inline assets/css/twenty-seventeen.scss]: Still now follow the coding standards.

Should be:

```scss
.colors-dark {
	.checkout-button {
		border: 2px solid #555;

		&:hover {
			border-color: #fff;
		}
	}
}
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]

File: includes/wc-template-functions.php (hunk 1 above)
Comment (maheshprajapati) [inline includes/wc-template-functions.php]: i think this is correct for this issue isset( $states ) && is_array( $states )

Comment (claudiosanches) [inline includes/wc-template-functions.php]: No idea for what issue.
And if is a different issue, so should be into a different Pull Request.

Comment (claudiulodro) [inline includes/wc-template-functions.php]: The CSS look good, but I can't merge this pull request until you remove this change. 

We do one pull request per issue, and this line doesn't have anything to do with the CSS changes. Feel free to open a separate PR with this change if there is a good reason for it, but I can tell you right now that it has some logic problems in it.

**Pull request closed without merging**